db/1.9/modules/ROOT/pages/database-connector-execute-ddl.adoc

```diff
@@ -0,0 +1,51 @@
+= Execute DDL Statements Example - Mule 4
+
+Anypoint Connector for Database (Database Connector) Execute DDL operation enables to execute DDL queries against a database. Data Definition Language (DDL) statements, are special types of SQL statements that do not directly access or modify data. Instead, they create, modify, or destroy data structures, stored procedures, and so on. You can also use DDL statements to create or modify data within tables or other data structures.
+
+
+== Configure the Execute DDL Operation in Studio
+
+ The following example shows how to configure the Execute DDL operation that creates a new table named `electronic` with columns named `id`, `name`, `description`, `price`, and `discount`:
```

Comment:
Could you check if there is some hidden format here since I'm seeing this text as `core`, image attached:
<img width="667" alt="Screen Shot 2021-02-10 at 6 03 14 PM" src="https://user-images.githubusercontent.com/59581166/107571948-5d6e2f00-6bca-11eb-905d-f67c4b14eb11.png">